src/views/Timeline.vue

```diff
@@ -22,99 +23,160 @@
-								<!-- <CheckboxRadioSwitch v-if="allowSelection"
-								class="selection-checkbox"
-								:checked="selectedSections[item.id]"
-								@update:checked="(value) => onSectionToggle(item.id)"> -->
-								<b>{{ item.id | dateMonth }}</b>
-								{{ item.id | dateYear }}
-								<!-- </CheckboxRadioSwitch> -->
-							</h3>
-
-							<File v-else
-								:item="files[item.id]"
-								:allow-selection="true"
-								:selected="selectedItems[item.id] === true"
-								:visibility="row.visibility"
-								:semaphore="semaphore"
-								@click="openViewer"
-								@select-toggled="onItemSelectToggle" />
-						</template>
-					</TiledRows>
-				</VirtualScrolling>
-			</TiledLayout>
-
-			<Loader v-if="loading" />
-
-			<EmptyContent v-if="isEmpty" illustration-name="empty">
-				{{ t('photos', 'No photos in here') }}
-			</EmptyContent>
+	<div v-else class="timeline">
+		<div class="timeline__header">
+			<Actions v-if="selectedFileIds.length === 0"
+				:force-title="true"
+				:force-menu="true"
+				:menu-title="t('photos', 'Add')"
+				:primary="true">
+				<Plus slot="icon" />
+				<ActionButton :close-after-click="true" :title="t('photos', 'Upload media')" @click="openUploader">
+					<FileUpload slot="icon" />
+				</ActionButton>
+				<ActionButton :close-after-click="true"
+					:aria-label="t('photos', 'Create a new album')"
+					:title="t('photos', 'Create new album')"
+					@click="showAlbumCreationForm = true">
+					<PlusBoxMultiple slot="icon" />
+				</ActionButton>
+			</Actions>
+
+			<template v-else>
+				<Button :close-after-click="true"
+					type="primary"
+					:aria-label="t('photos', 'Add selection to an album')"
+					@click="showAlbumPicker = true">
+					<template #icon>
+						<Plus slot="icon" />
+					</template>
+					{{ t('photos', 'Add to album') }}
+				</Button>
+				<Actions>
+					<ActionButton :close-after-click="true"
+						:aria-label="t('photos', 'Download selection')"
+						:title="t('photos', 'Download')"
+						@click="downloadSelection">
+						<Download slot="icon" />
+					</ActionButton>
+					<ActionButton v-if="shouldFavorite"
+						:close-after-click="true"
+						:aria-label="t('photos', 'Mark selection as favorite')"
+						:title="t('photos', 'Favorite')"
+						@click="favoriteSelection">
+						<Star slot="icon" />
+					</ActionButton>
+					<ActionButton v-else
+						:close-after-click="true"
+						:aria-label="t('photos', 'Remove selection from favorites')"
+						:title="t('photos', 'Remove from favorites')"
+						@click="unFavoriteSelection">
+						<Star slot="icon" />
+					</ActionButton>
+					<ActionButton :close-after-click="true"
+						:title="t('photos', 'Delete')"
+						:aria-label="t('photos', 'Delete selection')"
+						@click="deleteSelection">
+						<Delete slot="icon" />
+					</ActionButton>
+				</Actions>
+				<!-- HACK: Needed to make the above Actions work, no idea why be it is like that in the documentation. -->
```

Comment:
FIXME

Reply:
@skjnldsv Do you know if it is expected to need another <Actions> to make the first one work ?